### PR TITLE
2021-03-18-ralucarbc-javascript-methods-n-functions

### LINKED ISSUE #34: feat: sort and merge two arrays
* [x] create the sort and merge functionality
* [x] remove duplicates functionality
* [x] display it in the browser

### LINKED ISSUE #44: feat: difference of 2 cuboids regardless of which is bigger
* [x] create array's elements product functionality
* [x] make the absolute difference between the products functionality
* [x] display it in the browser

### LINKED ISSUE #46: feat: twice as old
* [x] create the functionality
* [x] display it in the browser

### PR DESCRIPTION
**Pull request by RalucaRbc**
this closes #34
this closes #44
this closes #46

### CODE REVIEW

File: web/ralucarbc/javascript/merge-2-sorted-arrays/merge-2-sorted-arrays.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const divFinal = document.querySelector("#final-result")
+const divRead = document.querySelector("#read-arrays")
+let arr5 = [1, 3, 5, 7, 9, 11, 12];
+let arr6 = [1, 2, 3, 4, 5, 10, 12];
+let array= [... new Set((arr5.concat(arr6)).sort((a, b) => a-b))]
```
Comment (marianzburlea) [inline web/ralucarbc/javascript/merge-2-sorted-arrays/merge-2-sorted-arrays.js]: a ramas a - b fara pauze

Comment (marianzburlea) [reply web/ralucarbc/javascript/merge-2-sorted-arrays/merge-2-sorted-arrays.js]: grrr

**Review by QuintyHH — APPROVED** [review]
💯

**Review by MariaTeo — APPROVED** [review]
nice work

**Review by Constantin-Matei — APPROVED** [review]
Cool :)

**Review by M00N89 — COMMENTED** [review]
molto brava e bella 👍🏻

File: web/ralucarbc/javascript/merge-2-sorted-arrays/merge-2-sorted-arrays.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const divFinal = document.querySelector("#final-result");
+const divRead = document.querySelector("#read-arrays");
+let arr5 = [1, 3, 5, 7, 9, 11, 12];
+let arr6 = [1, 2, 3, 4, 5, 10, 12];
+let array = [...new Set(arr5.concat(arr6).sort((a, b) => a - b))];
+divRead.textContent =
```
Comment (marianzburlea) [inline web/ralucarbc/javascript/merge-2-sorted-arrays/merge-2-sorted-arrays.js:6]: You can use backticks

**Review by marianzburlea — APPROVED** [review]
💯

**Review by BogdanNegura — APPROVED** [review]
👍

**Pull request merged**